internal/wasm/store.go

```diff
@@ -420,7 +412,7 @@ func (s *Store) instantiate(
 		if exitErr, ok := err.(*sys.ExitError); ok { // Don't wrap an exit error!
 			return nil, exitErr
 		} else if err != nil {
-			return nil, fmt.Errorf("start %s failed: %w", module.funcDesc(funcSection, funcIdx), err)
+			return nil, fmt.Errorf("start %s failed: %w", module.funcDesc(SectionIDFunction, funcIdx), err)
```

Comment:
curious if this line is covered in store_test.go

Reply:
yeah, what happened was it didn't test the error message difference before for SectionIDHostFunction. that's why changing this worked.